python/llm/example/CPU/Applications/streaming-llm/run_streaming_llama.py

```diff
@@ -45,6 +45,7 @@
 import torch
```

Comment:
this folder is for CPU example; GPU example should go to `python/llm/example/GPU`

Reply:
refactored